analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldNotOnlyReturnConstant.CSharp11.cs

```diff
@@ -3,6 +3,8 @@
     public interface IFoo
     {
         static abstract int GetValue();
+
+        static virtual int GetAnotherValue() { return 42; } // Compliant - interface methods are ignored
     }
 
```

Comment:
Maybe we could cover u8 strings (compliant, **runtime** constant) and also raw strings:

```csharp
    public string Get() => "hello"; // Noncompliant

   public string Get_Raw() => """
        "Forty-two", said Deep Thought, with infinite majesty and calm.
        """; 
    // Noncompliant @-3

    // ref: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/reference-types#utf-8-string-literals
    public ReadOnlySpan<byte> Get_Utf_8() => "hello"u8; // Compliant, utf-8 strings are runtime constants (represented as ReadOnlySpan<byte>)

```